### DISCUSSION TITLE
Changing default dark mode to light

### DISCUSSION THREAD

**Discussion by sasuadrian** [chakra-ui/chakra-ui#5017] [Q&A] [answered]
Hello!

I didn't found a solution for this problem. 
Code block.

index.tsx
`ReactDOM.render(
  <React.StrictMode>
    <ChakraProvider theme={theme} resetCSS>
        <ColorModeScript />
        <App />
    </ChakraProvider>
  </React.StrictMode>,
  document.getElementById('root')
);`

theme.tsx
`
const styles: Styles = {
  global: (props) => ({
    body: {
      color: mode('light', 'light')(props),
      bg: mode('light', 'light')(props)
    }
  })
};

const components = {
  Link: {
    // setup light/dark mode component defaults
    baseStyle: (props: GlobalStyleProps) => ({
      color: mode('blue.400', 'blue.300')(props),
      body: {
        color: mode('gray.800', 'whiteAlpha.900')(props),
        bg: mode('white', 'gray.900')(props)
      }
    })
  }
};

const theme = extendTheme({
  components,
  styles
});

export default theme
`

Just want to set, by default, the light theme.

Thanks!

**Comment by dmitriifrlv** [comment, marked as answer]
You should downgrade your `chakra-ui/react` to v.1.6.10 because of the [issue with v1.6.12](https://github.com/chakra-ui/chakra-ui/issues/4987), clean your `localStorage` and it should just work - [DEMO](https://codesandbox.io/s/light-mode-q6j9j?file=/src/App.js)

**Reply by sasuadrian** [reply]
Hello,
I just downgrade the `color-mode` and works fine. 

![image](https://user-images.githubusercontent.com/52112896/140812243-dac0e638-f98a-4c62-9aeb-21551a04af53.png)

Thanks! 🗡️
